### ISSUE TITLE
Binary operators in expressions

### ISSUE THREAD

**Issue by frxstrem** [kierenj/0x10c-DevKit#154]
I would like to be able to use these operators in expressions: AND (`A & B`), OR (`A | B`), XOR (`A ^ B`), NOT (`~A`) and bitwise shifts (`A << B` and `A >> B`). When used with defined constants, they would make it much easier to use bitmasks etc.

**Comment by kierenj** [comment]
Good idea, was expecting someone to ask for this sooner.  Will implement
